test.py

```diff
@@ -5,12 +5,15 @@
 import unittest
 import itertools
 
+import pandas as pd
+
 from beautifultable import BeautifulTable
 
 
 class TableOperationsTestCase(unittest.TestCase):
     def setUp(self):
         self.create_table()
+        self.create_dataframe()
```

Comment:
wouldn't this mean that `create_dataframe` would get called before all the unit tests even unrelated ones? maybe generate the dataframes within the specifc tests

Reply:
Done

Reply:
Let's not use the `_data` attribute in the unit tests. private variable so only meant to be used from internal modules.
`[list(row) for row in list(self.table._data)]` => `[list(row) for row in table.rows]`